spec.bs

```diff
@@ -2236,6 +2248,38 @@ To <dfn>encode trusted signals keys</dfn> given an [=ordered set=] of [=strings=
 
 </div>
 
+A <dfn>bidding signals per interest group data</dfn> is a [=struct=] with the following
+[=struct/items=]:
+
+<dl dfn-for="bidding signals per interest group data">
+  : <dfn>updateIfOlderThanMs</dfn>
+  :: Null or a {{double}}. If non-null, it is the [=duration=] in milliseconds that specifies that
+    the [=interest group=] should be [updated](#interest-group-updates) if it hasn't been joined or

```

Comment:
should? is it truly optional https://infra.spec.whatwg.org/#conformance?

Reply:
Well I assume the thing we're trying to say is that an attempt at updating will always (must) be made, with no guarantees as to how if the update will succeed or not, right?

Reply:
Meant to put this in this comment box:

Yes, we always attempt an update for eligible interest groups, even though the update attempt might fail, with the exception that if we navigate away before the auction completes, the update attempt won't happen. We do continue trying to update for up to 10 minutes after navigating away (I don't think we put the give up after 10 minutes part in the spec...maybe that should be spec'd, though probably in another PR?)

Reply:
So, do you think I should use "must" here?

Reply:
OK so if this is just describing what other parts of the spec already concretely do, then I think I would not use an RFC keyword at all here. I would say:

```
If non-null, it is the [=duration=] in milliseconds after which the [=interest group=] is eligible for an [update](#interest-group-update), if it hasn't been joined or updated within that duration. When eligible, the update is performed either following an auction by an interest group's buyers (linking somewhere would be good), or explicitly by the {{navigator/updateAdInterestGroups()}} method.
```

Reply:
Done.